aws/resource_aws_kinesis_firehose_delivery_stream.go

```diff
@@ -491,6 +492,22 @@ func resourceAwsKinesisFirehoseDeliveryStream() *schema.Resource {
 							Optional: true,
 						},
 
+						"s3_backup_mode": {
+							Type:     schema.TypeString,
+							Optional: true,
+							Default:  "Disabled",
```

Comment:
Minor nitpick: Prefer usage of the SDK provided constant `firehose.S3BackupModeDisabled`